pom.xml

```diff
@@ -10,7 +10,7 @@
   </parent>
 
   <artifactId>hiveberg</artifactId>
-  <version>1.0.0-SNAPSHOT</version>
+  <version>1.0.1-SNAPSHOT</version>
```

Comment:
Minor, but you can just leave this since we're not actually releasing anything yet.